### ISSUE TITLE
[Feature Request] Support for large epub files

### ISSUE THREAD

**Issue by alecdhansen** [5-stones/react-native-readium#66]
Maybe this is less of a request, and more of a question. What ways of going about handling large epubs without crashing your app? 200mb+

**Comment by jspizziri** [comment]
@alecdhansen yes, this is a question. Given that I have no idea how your app works I can't really help you here. I'm pretty sure I've opened epubs larger than that with this lib although I've never checked. If you have a public domain or free book that is larger than that I can test it with the example project (or you could) to make sure it works.

**Comment by alecdhansen** [comment]
@jspizziri I actually think my issue is more with react-native-fs than this lib. I don't have either of those resources available to share currently. 

Thanks for the quick response!
